.gitignore

```diff
@@ -3,3 +3,4 @@
 *.aar
 *.apk
 .vscode
+.idea
```

Comment:
I don't think this is necessary for this change.

Reply:
Oh yeah, this could probably be taken out. I added this to .gitignore because I was getting tired of seeing GoLand's garbage files in my diffs.

Reply:
Took it out in 481985a.

Reply:
This still remains. Could you remove this?

Reply:
Sorry this was removed.